### ISSUE TITLE
md5sum

### ISSUE THREAD

**Issue by Dekken** [CBPP/cbpp10-amd64#4]
Is the md5sum noted @ https://crunchbangplusplus.org of "97b14d55f41da080ed95e6c9a2a25ee2" for the ISO?

I've never gotten that sum from it either from the Unofficial HTTP or the torrent.

**Comment by Kurgol** [comment]
So the current md5sum on the homepage is generated for the 10.1 images. 

This is what I get for the old 10.0 releases, which as you say don't match the original md5's published. 

```
kurgol@JCN:/mnt/h/ISO/linux/cb++$ md5sum cbpp-10.0-amd64-20190707.iso
e5d4e1ad06805bb1dfcb55932b0db03e  cbpp-10.0-amd64-20190707.iso
kurgol@JCN:/mnt/h/ISO/linux/cb++$ md5sum cbpp-10.0-i686-20190707.iso
bd6451f0f83d3a31d21a9df814b1f9fd  cbpp-10.0-i686-20190707.iso
```

And this is what I get for the new 10.1 releases, which match what's currently on the downloads page
```
kurgol@JCN:/mnt/h/ISO/linux/cb++$ md5sum cbpp-10.1-
cbpp-10.1-amd64-20190713.iso  cbpp-10.1-i686-20190713.iso
kurgol@JCN:/mnt/h/ISO/linux/cb++$ md5sum cbpp-10.1-amd64-20190713.iso
0fdea701b5709a4f3c79f60da3c1b0f7  cbpp-10.1-amd64-20190713.iso
kurgol@JCN:/mnt/h/ISO/linux/cb++$ md5sum cbpp-10.1-i686-20190713.iso
c59549231f0fc0e85bc4f6ba111f5e01  cbpp-10.1-i686-20190713.iso
```

![image](https://user-images.githubusercontent.com/15245142/61383291-4b20f780-a8a6-11e9-858d-ccbb2284a837.png)

So probably worth grabbing the new 10.1 iso's as I'm getting them to match, and I doubt @computermouth  will fix the ones for 10.0 (if in fact they are broken, it's entirely possibly my download has a corruption)

**Comment by Kurgol** [comment]
The new 10.1 release are direct downloads hosted off github, so should be a faster download too =)
